glossary.asciidoc

```diff
@@ -65,14 +65,20 @@ DApp::
     Decentralised Application. At the very least, it is a smart contract and a web user-interface. More broadly, a DApp is a web application that is built on top of open, decentralized, peer-to-peer infrastructure services. In addition, many DApps include decentralized storage and/or message protocol and platform.
 
 DAO::
-  The hard fork at block #1,192,000 that reversed the hacked DAO contract and caused Ethereum and Ethereum Classic to split into two competing systems.
+  Decentralised Autonomous Organization. The hard fork at block #1,192,000 that reversed the hacked DAO contract and caused Ethereum and Ethereum Classic to split into two competing systems.
```

Comment:
We need to differentiate between "The DAO" (which was the hacked contract), DAO the term defining a Decentralized Autonomous Organization and DAO, the hard fork to fix the hack. All can go in this single definition. Start with the generic term, then The DAO contract, then the hard fork to fix the hack.

Reply:
sure